packages/gatsby-dev-cli/src/watch.js

```diff
@@ -8,21 +8,23 @@ let numCopied = 0
 const debouncedQuit = _.debounce(() => {
```

Comment:
Do we still need to debounce?

Reply:
I think so. I think the watcher gets triggered pretty often, and so the debounce is necessary so that we don't quit too soon.

That said, that's more of a guess than a firm guarantee 🤷‍♂️

Reply:
Oh I see... yeah, on `scan-once` we should only listen to the `ready` event then as that's only called once Chokidar has scanned all files https://github.com/paulmillr/chokidar

Reply:
Neat :) I'll fix this in a sec. Deep in caching stuff 😅